src/core/modes/walk-mode.ts

```diff
@@ -59,18 +59,37 @@ export class WalkMode implements Mode {
       return;
     }
 
-    const position = firstVisibleRange.start;
-    const range = new vscode.Range(position, position);
+    let visibleStartLine = firstVisibleRange.start;
+    let visibleEndLine = firstVisibleRange.start;
+    const junbaeLocation = { top: '5%', 'font-size': '50px' };
+    const location = vscode.workspace.getConfiguration('junbae-mode').get('location');
+
+    if (location === 'top') {
+      visibleStartLine = firstVisibleRange.start;
+      visibleEndLine = firstVisibleRange.start;
+    }
+    if (location === 'bottom') {
+      visibleStartLine = firstVisibleRange.end;
+      visibleEndLine = firstVisibleRange.end;
+      junbaeLocation.top = '-100px';
+      junbaeLocation['font-size'] = '100px';
```

Comment:
Why `font-size` different with top?

Reply:
When at the top, the spacing is different from when at the bottom, and the font size has an effect.

Reply:
you mean `font-size` determine spacing between bar and motions?

Reply:
answer plz